internal/pkg/remote/endpoint/service.go

```diff
@@ -188,22 +188,18 @@ func (ep *Config) GetAllServices() (map[string][]Service, error) {
 }
 
 // GetServiceURI returns the URI for the service at the specified SCS endpoint
-// Examples of services: consent, registry, key, token
+// Examples of services: consent, build, library, key, token
```

Comment:
@DrDaveD there is no longer a remote build

Reply:
Fixed